### ISSUE TITLE
min force arrows go below GFLBConstants.PULL_FORCE_RANGE.min

### ISSUE THREAD

**Issue by zepumph** [phetsims/gravity-force-lab-basics#120]
@terracoda and I found a potential bug where the visual min force is .7N, but the min force outlined in the constants in 35. It seems like those should be the same. I wonder if this is a bug.

**Comment by terracoda** [comment]
@zepumph I didn't see anything about a minimum force in the main design doc, just minimum mass and minimum distance.

minimum mass = 1 billion kg on each
minimum distance = 2.2 km (originally it was 2.5 km, but I think that was changed to 2.2 in our work and further refinement)

**Comment by terracoda** [comment]
Ooops, the minimum force would be a combo of min mass and max distance!

minimum mass = 1 billion kg on each
maximum distance = 9.6 km

With those numbers, is the min force 0.7 N?

**Comment by zepumph** [comment]
Yes! That is correct from my calculations. 

I think the way to proceed is to remove these "constants" because the usages of these should really be looking at the inputs (mass and distance) instead of assuming that the force (output) is always within that range. I haven't totally looked at usages of these, but hopefully we can use the model to output the min/max force based on constants like "MIN_MASS" and "MAX_DISTANCE." That feels like it will be the best long term solution.

**Comment by zepumph** [comment]
What if something like `ISLCModel.calculateForce` was static and public so that in the constants file we could use the ranges of distance/mass to actually calculate the correct constants for force within that file?

**Comment by zepumph** [comment]
There are 2 places where these constants are used in REGULAR and BASICS:

1. Force describers
2. For linear mappings of the puller images in PullerNode.js

**Comment by zepumph** [comment]
`ISLCModel` already had two prototype methods: `getMaxForce`/`getMinForce`.

Everywhere we were using the force constants we were able to easily access an instance of the ISLCModel, so we could just use those methods instead.


Also while I was add it, I converted the left/right boundary constants into a range also, for clarity. 


I'm happy with how much duplicated, assuming code I just removed. I think that this change is definitely for the better for all three sims.
